senior_game_client/src/ui/setup.rs

```diff
@@ -0,0 +1,52 @@
+use bevy::prelude::*;
+use bevy_egui::{egui, EguiContext};
+
+pub fn setup_ui(mut egui_ctx: ResMut<EguiContext>) {
+  let ctx = &mut egui_ctx.ctx;
+
+  let mut fonts = egui::FontDefinitions::default();
+
+  fonts.font_data.insert(
+    "LibreBaskerville-Regular".to_owned(),
+    std::borrow::Cow::Borrowed(include_bytes!(
+      "../../assets/fonts/LibreBaskerville-Regular.ttf"
```

Comment:
I think using one line for this instead of two makes it a little more readable and is consistent with the other font_data.insert below this one